formats/polus-tabular-to-feather-plugin/src/main.py

```diff
@@ -0,0 +1,128 @@
+from bfio.bfio import BioReader, BioWriter
+from pathlib import Path
+import fcsparser
+import os
+import argparse
+import logging
+import vaex
+
+# Import environment variables
+POLUS_LOG = getattr(logging,os.environ.get('POLUS_LOG','INFO'))
+POLUS_EXT = os.environ.get('POLUS_EXT','.ome.tif')
+
+# Initialize the logger
+logging.basicConfig(format='%(asctime)s - %(name)-8s - %(levelname)-8s - %(message)s',
+                    datefmt='%d-%b-%y %H:%M:%S')
+logger = logging.getLogger("main")
+logger.setLevel(POLUS_LOG)
+
+def fcs_csv(file,outDir):
+        """Convert fcs file to csv. Copied from polus-fcs-to-csv-converter plugin.
+            
+            Args:
+                file (str): Path to the directory containing the fcs file.
+                outDir (str): Path to save the output csv file.
+                
+            Returns:
+                Converted csv file.
+                
+        """
+        file_name = Path(file).stem
+        logger.info('Started converting the fcs file ' + file_name)
+        meta, data = fcsparser.parse(file, meta_data_only=False, reformat_meta=True)
+        logger.info('Saving csv file ' + file_name)
+        #Export the file as csv
+        os.chdir(outDir)
+        export_csv = data.to_csv (r'%s.csv'%file_name, index = None, header=True,encoding='utf-8-sig')
+        return export_csv
+    
+def df_toFeather(file,outDir):
+        """Convert a csv or fcs file to Arrow feather file. Includes code from polus-fcs-to-csv-converter plugin
+                
+            Args:
+                file (str): Path to the directory containing the csv file.
+                outDir (str): Path to the directory to save feather file.
+                    
+            Returns:
+                Feather format file.
+                    
+            """
+        file_name = Path(file).stem
+        feather_filename = file_name + '.feather'
+        
+        logger.info('Started converting the csv file ' + file_name)
+        df = vaex.from_csv(file)
```

Comment:
This doesn't seem correct. Shouldn't there be a switch here?

Also, I seem to remember using hdf5 as an intermediate to improve scalability in the conversion process.

One thing that seems to be missing is the `vaex.open` function that should be used for opening hdf5, parquet, etc files.